content/docs/1.0.0/tutorial/get-to-know-iofog.md

```diff
@@ -86,36 +82,28 @@ iofog-agent info
   Fog type                                 : intel_amd
 ```
 
-When you're ready to leave the current container, simply use `exit`.
-
-```sh
-exit
-```
-
-This tutorial has another Agent too, `iofog-agent-2`, you might check it out as well!
-
 <aside class="notifications note">
   <h3><img src="/images/icos/ico-note.svg" alt=""> Learn more Agent commands</h3>
   <p>If you want to learn about all the possible commands you can use on iofog-agent checkout the <a href="../agents/cli-usage.html">CLI Usage guide</a>.</p>
 </aside>
 
 #### Controller's Container
 
-After we've exited from our Agent's containers, let's start a shell into the Controller's container:
-
-```sh
-docker exec -ti iofog-controller bash
+Let's try now listing all the preconfigured ioFog nodes using the Controller CLI.
+
+```console
+$ docker exec -ti iofog-controller iofog-controller iofog list
+{
+  "fogs": [
+    {
+      "uuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+      "name": "Default Fog",
+      "location": null,
+      "gpsMode": "off",
+...
 ```
 
-Since this "device" is running our Controller daemon, inside this shell we can now issue commands to it. Try this one:
-
-```sh
-iofog-controller iofog list
-```
-
-This should give you a JSON response containing a list of edge nodes already registered with the Controller. In our case we should get two nodes, `"name": "Agent 1"` and `"name": "Agent 2"`. These "nodes" are the container environments iofog-agent-1 and iofog-agent-2. Remember that in production, instead of being containers running locally on your machine, these nodes would be real edge hardware, part of your Edge Compute Network, where your microservices run.
-
-When you want to exit that session, you can use the `exit` command again.
+This should give you a JSON response containing a list of edge nodes already registered with the Controller. In our case we should get a single node, `"name": "Default Fog"`. This "node" is the container environment of iofog-agent. Remember that in production, instead of being containers running locally on your machine, these nodes would be real edge hardware, part of your Edge Compute Network, where your microservices run.
```

Comment:
I don't think a comma is necessary after "real edge hardware" unless you're saying: 

...these nodes would be real edge hardware **as a part of our Edge Compute Network (ECN), where our microservices run.**